kenware/Assessment
# #assessment event endpoints

#### What does this PR do?
- create assessment event endpoints

#### Description
- the endpoint to start assessment
- the endpoint to post an answer to a questions
- the endpoint to submit assessment
- add test
#### How should this be manually tested?
- Create a local copy `git fetch origin create-start-assessment-endpoint`
- Checkout to the branch `git checkout create-start-assessment-endpoint`
- Start the application with the command: `python manage.py runserver`
- On postman to start assessment make a GET request to `{{base-url}}/api/v1/assessments/event/?assessmentId=2&include=children`
- On postman to answer a question make a POST request to `{{base-url}}/api/v1/assessments/event/`
- An authenticated user should be able to carry out all crude operation on user endpoints
- To run tests use `python manage.py test`
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#162187176](https://www.pivotaltracker.com/story/show/162187176)
